pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/v3/AdminApiTransactionTest.java

```diff
@@ -396,6 +396,7 @@ private static void verifyCoordinatorStats(String state,
     private void initTransaction(int coordinatorSize) throws Exception {
         admin.topics().createPartitionedTopic(TopicName.TRANSACTION_COORDINATOR_ASSIGN.toString(), coordinatorSize);
         admin.lookups().lookupTopic(TopicName.TRANSACTION_COORDINATOR_ASSIGN.toString());
+        pulsarClient = PulsarClient.builder().serviceUrl(lookupUrl.toString()).enableTransaction(true).build();
```

Comment:
please close this client, otherwise the new variable below will override the reference and we will leak a PulsarClient